em_workflows/lrg_2d_rgb/flow.py

```diff
@@ -155,7 +163,10 @@ def lrg_2d_flow(
     -create tmp dir for each.
     -convert to tiff -> zarr -> jpegs (thumb)
     """
-    utils.notify_api_running(no_api, token, callback_url)
+    if no_api:
+        utils.notify_api_running(no_api=no_api)
+    else:
+        utils.notify_api_running(token=token, callback_url=callback_url)
```

Comment:
If `no_api=True, token=<valid token>, callback_url=<valid url>`, the first if statement will be used and notify function will be called without any token or cb_url. I still believe the original function definition (of `notify_api_running`) is still better.